Gemfile.jruby-2.5.lock.release

```diff
@@ -516,9 +515,9 @@ GEM
     lru_redux (1.1.0)
     mail (2.6.6)
       mime-types (>= 1.16, < 4)
-    manticore (0.6.4-java)
+    manticore (0.7.0-java)
```

Comment:
- Fix a thread safety issue with regards to adding requests to the parallel execution queue while the client is already processing a queue  
- drop support for Ruby 1.7